docs/class-definitions.md

```diff
@@ -386,38 +386,41 @@ Handles tracking of member cursors within a space. Inherits from [EventEmitter](
 
 ## Methods
 
-### get
+### set
+
+Set a `CursorUpdate`. If a member has not yet entered the space, this method will error.
```

Comment:
```suggestion
Set the position of a cursor. This will emit a `CursorUpdate` event. If a member has not yet entered the space, this method will error.
```